front/app/modules/commercial/moderation/admin/containers/ModerationRow.tsx

```diff
@@ -72,8 +73,8 @@ const GoToLink = styled(Link)`
 `;
 
 const GoToIcon = styled(Icon)`
-  width: 100%;
-  height: 100%;
+  width: 14px;
```

Comment:
This will probably be too small with the new icons. Can we leave it without a custom size?

Reply:
Ben also said this was too small, fixed it :+1: